### ISSUE TITLE
Support module loaders

### ISSUE THREAD

**Issue by mpschaeuble** [w8r/Leaflet.Path.Drag#35]
If Leaflet.Path.Drag is used with a module system, a packager like webpack cannot resolve the dependencies if this plugin has no support for it. As a result, the plugin cannot be used as the script loading order cannot be determined.

Please add support for AMD and Node/CommonJS module systems.

**Comment by w8r** [comment]
Of course, the library supports them, it's just that Webpack3 cannot read it properly. I will fix it.

**Comment by davidbeers** [comment]
Any progress on this, @w8r? Leaflet.Path.Drag is a plugin I would really like to be able to use, but I am prevented by not being able to use it as a module.
